lib/param_parsing/terraform_version_test.go

```diff
@@ -0,0 +1,15 @@
+package param_parsing
+
+import (
+	"testing"
+)
+
+func TestGetParamsFromTerraformVersion(t *testing.T) {
```

Comment:
Can there be a test for running against a non-existent file?

Reply:
I suspect this applies to the other parser testing that have the same functionality

Reply:
added various test cases for that